### ISSUE TITLE
StackStorm v3.4.0 Pre-release Testing

### ISSUE THREAD

**Issue by blag** [StackStorm/discussions#61]
> We're not yet ready to prepare the StackStorm v3.4 release and start the pre-release testing, but I'm opening this issue now in preparation. We still need to include a list of major features or areas that saw a large amount of code churn, as well as a complete list of the changes from the changelog so users can test each change. See the [previous pre-release testing discussion](https://github.com/StackStorm/discussions/issues/52) for an example.

# Release Process Preparation

Per [Release Management Schedule](https://github.com/StackStorm/discussions/wiki/Release-Management-Rotation) @blag  is the Release Manager and @amanda11 is Assisting for v3.4. They will freeze the `master` for the major repositories in StackStorm org, follow the [StackStorm Release Process](https://github.com/StackStorm/discussions/wiki/Release-Process) which is now available to public, accompanied by the [Useful Info for Release managers](https://github.com/StackStorm/discussions/wiki/Useful-info-for-Release-Managers). Communication is happening in `#releasemgmt` and `#development` Slack channels.
The first step is pre-release manual user-acceptance testing for `v3.4dev`.

# Why Manual testing?

StackStorm is very serious about testing and has a lot of it:

* [Linting, Compile, and Unit tests](https://github.com/StackStorm/st2/actions?query=branch%3Amaster)
* [Integration tests](https://travis-ci.org/github/StackStorm/st2)
* [User Interface tests](https://app.circleci.com/pipelines/github/StackStorm/st2web)
* [Packaging tests](https://app.circleci.com/pipelines/github/StackStorm/st2)
* [Deployment Integrity checks](https://github.com/StackStorm/st2-packages/tree/master/rake/spec/default)
* [Smoke tests](https://github.com/StackStorm/ansible-st2/tree/master/roles/StackStorm.st2smoketests/tasks)
* end-to-end tests when automation spins up new AWS instance for each OS/flavor we support, installs real st2 like user would and runs set of [st2tests](https://github.com/stackstorm/st2tests) (for each st2 PR, nightly, periodically, during release), including end-to-end tests ChatOps

> See [st2ci](https://github.com/stackstorm/st2ci) and [st2cd](https://github.com/stackstorm/st2cd) for more examples and workflows about how StackStorm automation is used to test StackStorm (dogfooding).

That's a perfect way to verify what we already know and codify expectations about how StackStorm should function. 

However it's not enough.
There are always new unknowns to discover, edge cases to experience and tests to add. Hence, manual [Exploratory Testing](https://en.wikipedia.org/wiki/Exploratory_testing) is an exercise where entire team gathers together and starts trying (or breaking) new features before the new release. Because we're all different, perceive software differently and try different things we might find new bugs, improper design, oversights, edge cases and more.

This is how StackStorm previously managed to land less major/critical bugs into production.

# TL;DR

See the [Testing Process](https://github.com/StackStorm/discussions/wiki/Release-Process#Testing-Process), where it will walk you through:

* Spinning up a StackStorm instance using st2vagrant
* Running StackStorm's self-check script
* Running through the basic manual testing steps

Additionally, try to use that StackStorm instance as you normally would, maybe try to break it in new and interesting ways that you haven't tried before, and report any regressions found comparing to `v3.3`.

Extra points for PR hotfixes, reporting entirely new bugs, and missing test cases!

## Specific things to check ##

* [ ] Proper integration between st2web and st2flow
  - [ ] The blue `+` button to create a new workflow is visible in the actions list in the `Actions` tab
  - [ ] An action that is an Orquesta workflow has a blue `EDIT` button in the action details pane
* [ ] Python 2 deprecation
  - [ ] ST2 should install packs with Python 3.6 by default
  - [ ] The `python3` boolean option should not appear in the web UI in the `packs.install` action
  - [ ] The `st2 pack install` command should print a warning when passed the `--python3` flag
  - [ ] New sensors are run with Python 3.6
  - [ ] Installation on U16 requires a PPA. A warning and confirmation will be required before it is installed by the ST2 installers, but flags/variables are available to automatically install as well.
* [ ] st2resultstracker was removed
  - [ ] `st2ctl status` does not show any running `st2resultstracker` processes
  - [ ] `st2ctl restart` does not start any `st2resultstracker` processes
* [ ] RBAC
  - [ ] RBAC not enabled by default by installers
  - [ ] RBAC can be enabled by following latest documentation
* [ ] LDAP
* [ ] Add support for GitLab SSH URLs

**Comment by arm4b** [comment]
Duplicate of https://github.com/StackStorm/discussions/issues/66
